tests/test_sota_checkpoints.py

```diff
@@ -498,6 +498,13 @@ def openvino_preinstall(openvino):
                        check=True, shell=True)
 
 
+@pytest.fixture(autouse=True, scope="class")
+def clean_previous_metrics_dump_dir():
+    if os.path.isdir(METRICS_DUMP_PATH):
+        subprocess.run("rm -f *.json", cwd=METRICS_DUMP_PATH, check=True, shell=True)
```

Comment:
It does not look like a safe method and is unlikely to pass the SDL checks.  The second concern, that it is not a portable solution. What do you think about using the standard module, for example, `shutil`?

Reply:
@pfinashx, `"rm -rf *"` looks scary indeed. I also think that the `os.` approach should be used, just so that the test invocation does not accidentally use '/' as the current working directory. 

@daniil-lyakhov with the `os` approach, should either filter for `.json` files or delete the directory and recreate it instead of removing all files inside it.

Reply:
Completely agree with that. I used to remove only `.json` files because test keeps log dir inside `metrics_dump` dir. Shall we remove all previous run logs before new launch?

Reply:
If subprocess does not safe (although more safe file mask can be used, like "*.metrics.json")
We can use construction with glob.glob to find files and os.remove to delete they - 
`for file in glob.glob(f'{METRICS_DUMP_PATH}/*metrics.json'):
    os.remove(file)`

Reply:
Please run NNCF eval with your branch and add job number here.

Reply:
Could you please provide job link?

Reply:
I started job #30 on Ubuntu20